### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by tasleson**
Some misc. cleanup and some targeted changes.  More to come with regards to py3 compatibility possibility with adding the ability to return target ports.

### CODE REVIEW

**Comment by tasleson** [comment]
@cathay4t Please give this a review when you get a chance, thanks!

**Comment by cathay4t** [comment]
The patches looks good to me. Let me perform some tests before we get this merged.

File: targetd/main.py [hunk 1]
```diff
@@ -158,6 +159,35 @@ def finish_request(self, sock, addr):
...
+            if stat.S_ISREG(ss.st_mode):
+                if ss.st_uid == 0:
+                    if ss.st_mode & 0077 == 0 and \
+                            bool(ss.st_mode & stat.S_IRUSR):
+                        rc = True
+                    else:
+                        log.error("SSL file: '%s' incorrect permissions (%s), "
+                                  "ensure file readable or writable by anyone "
+                                  "other than owner, and that owner "
+                                  "can read." % (f, oct(ss.st_mode & 0777)))
```
Comment (cathay4t) [inline targetd/main.py]: Should be `ensure file is _not_ readable or writeable by anyone other than owner, and that owner can read.`.

Or even include suggested mode code `0400`.

**Comment by cathay4t** [comment]
Besides a incorrect warning message, this patch set is tested.

**Comment by tasleson** [comment]
@cathay4t Thanks!, I will reword error message

**Comment by tasleson** [comment]
@cathay4t Error message corrected with b26a8d7c64bef011d6300c69140fef0f0a3cce95

**Comment by cathay4t** [comment]
@tasleson I failed to find b26a8d7 other than github webpage, could you include it in this PR?

**Comment by tasleson** [comment]
@cathay4t OK, I edited in place and did a forced update of this branch to make the error message change.

**Comment by cathay4t** [comment]
Merged. Thanks.

**Pull request merged**